ceph-ansible-prs/config/definitions/ceph-ansible-prs.yml

```diff
@@ -24,11 +40,78 @@
       - ansible2.2-bluestore_dmcrypt_journal
       - ansible2.2-bluestore_dmcrypt_journal_collocation
     jobs:
-        - 'ceph-ansible-prs-{release}-{scenario}'
+        - 'ceph-ansible-prs-{release_before_luminous}-{scenario_before_luminous}'
+        - 'ceph-ansible-prs-{release_after_luminous}-{scenario_after_luminous}'
+
+- job-template:
+    name: 'ceph-ansible-prs-{release_before_luminous}-{scenario_before_luminous}'
```

Comment:
We did that https://github.com/ceph/ceph-ansible/pull/1524/commits/80c82f0950e3c9f624f779f425fbfa3e42555b48 but the CI is still testing some scenarios.
So we thought we had to change something here.